test/browser_tests/page_objects/base-page.js

```diff
@@ -1,4 +1,4 @@
-const { setDefaultTimeout } = require( 'cucumber' );
+const { setDefaultTimeout } = require( '@cucumber/cucumber' );
```

Comment:
Should have been updated in https://github.com/cfpb/consumerfinance.gov/pull/6317 oooops